zebra-state/src/in_memory.rs

```diff
@@ -105,6 +105,7 @@ impl Service<Request> for InMemoryState {
 
                 async move { Ok(Response::BlockLocator { block_locator }) }.boxed()
             }
+            Request::GetUTXO { .. } => todo!(),
```

Comment:
It might just be easier to delete InMemoryState, and use a temporary sled state for testing.

Reply:
I agree, there was a brief discussion about this on Discord (with the idea that the service transcript functionality we have now mostly serves the use-case that the in-memory backend was intended for).

Reply:
totally onboard with this, kill the in memory service!

Reply:
#864 :)